space2stats_api/src/tests/conftest.py

```diff
@@ -28,10 +31,50 @@ def s3_client(aws_credentials):
 def test_bucket(s3_client) -> str:
     bucket_name = "test-bucket"
     s3_client.create_bucket(Bucket=bucket_name)
-
     return bucket_name
 
 
+@pytest.fixture(scope="session")
+def database(postgresql_proc):
+    with DatabaseJanitor(
+        user=postgresql_proc.user,
+        host=postgresql_proc.host,
+        port=postgresql_proc.port,
+        dbname="testdb",
+        version=postgresql_proc.version,
+        password="password",
+    ) as jan:
+        db_url = (
+            f"postgresql://{jan.user}:{jan.password}@{jan.host}:{jan.port}/{jan.dbname}"
+        )
+        with psycopg.connect(db_url) as conn:
+            with conn.cursor() as cur:
+                cur.execute("""
+                    CREATE TABLE IF NOT EXISTS space2stats (
+                        hex_id TEXT PRIMARY KEY,
+                        sum_pop_2020 INT,
+                        sum_pop_f_10_2020 INT
+                    );
+                """)
+                cur.execute("""
+                    INSERT INTO space2stats (hex_id, sum_pop_2020, sum_pop_f_10_2020)
+                    VALUES ('hex_1', 100, 200), ('hex_2', 150, 250);
+                """)
+
+        yield jan
```

Comment:
moving the database configuration in one block

Reply:
nice!